### ISSUE TITLE
The version 0.11 is not recognized with SPM.

### ISSUE THREAD

**Issue by mitsuse** [Carthage/Commandant#75]
The version string of patch seems to be required in the current implementation of SPM. I executed `swift package update` with [this manifest file][Package.swift] and got the output as follow:

```
Cloning https://github.com/Carthage/Commandant.git
error: The dependency graph could not be satisfied. The package (https://github.com/Carthage/Commandant.git) with version tag in range (0.11.0..<0.11.9223372036854775807) is not found. Found tags ([0.1.1, 0.6.1, 0.7.0-beta.1, 0.7.0, 0.8.1, 0.8.2, 0.8.3, 0.9.0, 0.10.0, 0.10.1])
```

They are related source codes to this issue:

- https://github.com/apple/swift-package-manager/blob/swift-3.0-RELEASE/Sources/PackageDescription/Version+StringLiteralConvertible.swift#L40-L46
- https://github.com/apple/swift-package-manager/blob/swift-3.0-RELEASE/Sources/Utility/Git.swift#L78-L83

[Package.swift]: https://gist.github.com/mitsuse/639bac06675e909dc66e6134705fba1b

**Comment by ikesyo** [comment]
Looks like `0.11` is not listed in the found tags messeage:

> Found tags ([0.1.1, 0.6.1, 0.7.0-beta.1, 0.7.0, 0.8.1, 0.8.2, 0.8.3, 0.9.0, 0.10.0, 0.10.1])

You might have to run `swift package fetch` or `swift package update`.

**Comment by mitsuse** [comment]
Thanks for a quick reply. I tried also `swift package fetch` but got a same message as before.

**Comment by ikesyo** [comment]
That's really annoying...... 😫

**Comment by ikesyo** [comment]
Okay, we should consistently use `x.y.z` style tag to meet Semantic Versioning spec: http://semver.org/#spec-item-2

> A normal version number MUST take the form X.Y.Z where X, Y, and Z are non-negative integers, and MUST NOT contain leading zeroes. X is the major version, Y is the minor version, and Z is the patch version. Each element MUST increase numerically. For instance: 1.9.0 -> 1.10.0 -> 1.11.0.

@mdiep I'm going to cut 0.11.1 release to just fix the versioning issue.

**Comment by mitsuse** [comment]
Thanks!

**Comment by ikesyo** [comment]
@mitsuse Thank you! This should be fixed with [0.11.1](https://github.com/Carthage/Commandant/releases/tag/0.11.1).

**Comment by mdiep** [comment]
Thanks @ikesyo!
